indexeddb-examples/README.md

```diff
@@ -0,0 +1,8 @@
+# indexeddb-examples
+Code examples that accompany the MDN IndexedDB documentation – https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API
+
+The "idbcursor" directory contains a very simple IndexedDB example to demonstrate the usage of IDBCursor. See [https://developer.mozilla.org/en-US/docs/Web/API/IDBCursor](https://developer.mozilla.org/en-US/docs/Web/API/IDBCursor) for relevant reference pages. [View the example live](http://mdn.github.io/indexeddb-examples/idbcursor/).
```

Comment:
I assume this will change to:

```suggestion
The "idbcursor" directory contains a very simple IndexedDB example to demonstrate the usage of IDBCursor. See [https://developer.mozilla.org/en-US/docs/Web/API/IDBCursor](https://developer.mozilla.org/en-US/docs/Web/API/IDBCursor) for relevant reference pages. [View the example live](http://mdn.github.io/dom-examples/indexeddb-examples/idbcursor/).
```